pkg/ifc/transformer/transformer.go

```diff
@@ -25,4 +25,5 @@ import (
 // TransformerFactory makes patch transformer.
 type TransformerFactory interface {
```

Comment:
ditto, if you want to put this a 'transformer' package, don't need Transformer in the class name - just call it Factory

FWIW, the transformerconfig package is slightly odd with its naming.
if desired, you could try moving it to pkg/transformer/config then shorten the object names in a different PR.